### PR TITLE
add time keys for NWP models

### PR DESCRIPTION
**Pull request by RousseauLambertLP**
add time keys for NWP models

And yes I forgot but we need the `self.new_key_store = False` because I'm using that key in the handler script to trigger the writting of the time keywords in Redis whenever a model run is complete, or when a new radar file is added. This is done in the count function of the layer/base.py script.

### CODE REVIEW

**Comment by RousseauLambertLP** [comment]
Note that I will implement the time information keys in Redis for Cansips and REPS in separated merge requests to avoid a big bulk pull request.

File: geomet_data_registry/layer/base.py [hunk 1]
```diff
@@ -160,16 +159,18 @@ def update_count(self, item, r, item_dict):
...
             elif int(new_layer_file_count) == 1:
                 for mr in self.model_run_list:
-                    layer_count_key_reset = '{}_{}_count'.format(
-                        item_dict['properties']['layer'], mr)
-                    mr_nm = int(self.store.get_key(layer_count_key_reset))
-                    if layer_count_key_reset != layer_count_key and mr_nm != 0:
+                    layer_count_key_reset = '{}_{}_{}_count'.format(
+                        self.model, self.wx_variable, mr)
+                    mr_nm = self.store.get_key(layer_count_key_reset)
+                    if layer_count_key_reset != layer_count_key and \
```
Comment (tomkralidis) [inline geomet_data_registry/layer/base.py]: Better written as:
```python
                    if layer_count_key_reset != layer_count_key and mr_nm not in ['0', None]:
```

Shouldn't `mr_nm` be an int (not `'0'`)?

File: geomet_data_registry/layer/cansips.py [hunk 2]
```diff
@@ -150,5 +150,17 @@ def identify(self, filepath):
...
+        TO BE DONE
+
+        Add time keys when applicable:
+            - model run default time
+            - model run extent
+            - forecast hour extent
+        and for observation:
+            - latest time step
+        """
+
```
Comment (tomkralidis) [inline geomet_data_registry/layer/cansips.py]: 1. these should not be blank (add `pass` at least)
2. Mark the section as `TODO` per https://www.python.org/dev/peps/pep-0350/

File: geomet_data_registry/layer/reps.py [hunk 3]
```diff
@@ -156,5 +156,17 @@ def identify(self, filepath):
...
+        TO BE DONE
+
+        Add time keys when applicable:
+            - model run default time
+            - model run extent
+            - forecast hour extent
+        and for observation:
+            - latest time step
+        """
+
```
Comment (tomkralidis) [inline geomet_data_registry/layer/reps.py]: See previous comment.

File: geomet_data_registry/layer/base.py (hunk 1 above)
Comment (RousseauLambertLP) [reply geomet_data_registry/layer/base.py]: yeah I remember now why I used that: 

we have: 
```
mr_nm = self.store.get_key(layer_count_key_reset)
```
Here when we have a cold start, `mr_nm = None` nad then I can't None to an integer: 
```
mr_nm = int(self.store.get_key(layer_count_key_reset))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

Comment (RousseauLambertLP) [reply geomet_data_registry/layer/base.py]: still I have updated the if statement like you suggested: `...and mr_nm not in ['0', None]:`

File: geomet_data_registry/layer/cansips.py (hunk 2 above)
Comment (RousseauLambertLP) [reply geomet_data_registry/layer/cansips.py]: added #TODO and pass

File: geomet_data_registry/layer/reps.py (hunk 3 above)
Comment (RousseauLambertLP) [reply geomet_data_registry/layer/reps.py]: added #TODO and pass

**Pull request merged**